### PR TITLE
add timing info for flx creation

### PR DESCRIPTION
**Pull request by devoncarew**
@mpcomplete, add some timing info for flx creation. I'm seeing roughly 600ms to call `CipherParameters.get().seedRandom()`, and about 1000ms to create the zip (via the `new ZipEncoder().encode(archive)` call). For comparison, shelling out to `zip` takes ~75ms.

### CODE REVIEW

File: packages/flx/test/signing_test.dart [hunk 1]
```diff
@@ -34,7 +34,7 @@ main() async {
 
   // Set up a key generator.
   CipherParameters cipher = CipherParameters.get();
-  await cipher.seedRandom();
+  cipher.seedRandom();
```
Comment (eseidelGoogle) [inline packages/flx/test/signing_test.dart]: Will this make it racey to not wait here?

Comment (mpcomplete) [inline packages/flx/test/signing_test.dart]: This is not correct. The tests will sometimes run before we seed the RNG. Is it too slow? I can add a way to seed with some in-memory numbers - though that will get rid of the basic fuzz testing we had.

Comment (mpcomplete) [inline packages/flx/test/signing_test.dart]: Wait, ignore this. I somehow missed the change to make seedRandom sync.

Comment (devoncarew) [inline packages/flx/test/signing_test.dart]: Yeah, it was just async to read a few bytes from a file, and that can be done just as easily w/ sync read. I generally prefer `fooSync()` to `await fooAsync()` when a sync version of a method is available.

**Comment by devoncarew** [comment]
I was doing some basic profiling of our startup time (`flutter -v run`), and the random seeding and zip creation stood out. We should probably look for some places to streamline the startup.

**Comment by mpcomplete** [comment]
LGTM.

I'm pretty surprised by those numbers. 600ms to get a seed is bad, but 1000ms to create a zip is absurd.

Seeding is only used for signing the flx, and if we decide we don't need an updater, maybe it can be removed.

**Comment by devoncarew** [comment]
Yeah, I think there's some low-hanging fruit here, to knock 20-30% off our start times.

**Pull request merged**